README.md

```diff
@@ -43,7 +43,12 @@ Optionally, you could add a [git hook for post-checkout](https://ttboj.wordpress
 ### Project Settings
 If you plan on making changes to EditorXR and/or contributing back, then you'll need to set the `Asset Serialization` property under Edit->Project Settings->Editor to `Force Text`.
 
-We're using `#if UNITY_EDITOR` for now in order to keep our code out of your player builds. We will eventually explore a possible overlap between EditorXR and player builds. We recommend you do the same for your tools if you plan to distribute them to others.
+### Assembly Definitions
+In order to support a variety of platform configurations, and to optionally strip its code out of player builds, EditorXR uses assembly definitions. The EditorXR assembly definition must reference both platform SDK packages (SteamVR or Oculus VR) from the Asset Store. These packages do not include assembly definitions in their current forms, so after importing EditorXR, you must add them.
+
+For easy set-up, EditorXR includes a .unitypackage (`Patches/Dependencies_asmdef.unitypackage`) containing these assembly definitions in the locations they are likely to be needed. This package also contains an assembly definition for the PolyToolkit, which is also referenced by EditorXR.
+
+This is not required for Unity versions 2019.1 and above.

```

Comment:
Have you tested this?

Reply:
Yep! I tested it in one of my projects that was already mostly set up. One of my tasks for today is to do a clean setup from an empty project, both from github and from a package.